conftest.py

```
@@ -15,19 +15,35 @@
 import os
 
-import cupy as cp
 import numpy as np
 import pytest
-from holoscan.core import Application, Fragment
+
+
+def _import_cupy():
+    try:
+        import cupy as cp
+    except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)